### PR TITLE
Add emit destroy popover

### PR DESCRIPTION
**Pull request by kolesoffac**
It need when run action after end animation(hide).

Thanks

### CODE REVIEW

File: packages/popover/src/main.vue [hunk 1]
```diff
@@ -120,6 +120,10 @@ export default {
   },
 
   methods: {
+    doPrivateDestroy() {
+      this.doDestroy();
+      this.$emit('distroyed');
```
Comment (syn-zeta) [inline packages/popover/src/main.vue]: `destroyed`. The documentation will also need to be updated.

Can you rely on the existing `hide` event instead (emitted on line 64 above)? Why is this needed?

Comment (kolesoffac) [reply packages/popover/src/main.vue]: *Hide* triggered immediate, when transition don't finish yet! But sometimes need call action after full hide *popover*.

**Comment by Leopoldthecoder** [comment]
@kolesoffac Have you tried `hide` event?

File: packages/popover/src/main.vue (hunk 1 above)
Comment (kolesoffac) [reply packages/popover/src/main.vue]: >>The documentation will also need to be updated.

But where?

Comment (Leopoldthecoder) [reply packages/popover/src/main.vue]: @kolesoffac It's [here](https://github.com/ElemeFE/element/blob/dev/examples/docs/en-US/popover.md).

**Comment by Leopoldthecoder** [comment]
@kolesoffac If that's the case, I'm afraid its counterpart in the entering transition is also needed. Perhaps we can call them `after-enter` and `after-leave`.

If you're interested, you can update this PR.

**Comment by kolesoffac** [comment]
Are you interest and enter, and leave of transition?

**Comment by syn-zeta** [comment]
@kolesoffac Yes, it'd be nice to see both `after-enter` and `after-leave` events, in that case. There's also documentation in three languages, so you can also just copy-paste English description in other ones, and someone else will translate them (they'll act as placeholders until then, and the docs won't get out-of-sync).

**Comment by Leopoldthecoder** [comment]
I'll add `after-enter` and `after-leave` events right away. Closing this PR.

**Comment by Leopoldthecoder** [comment]
#10047
